.github/CONTRIBUTING.md

```diff
@@ -8,4 +8,21 @@ Following these guidelines will help make sure that users and developers alike c
 
 Please do NOT post any copyrighted materials. This includes (but is not limited to) ROMs, ROM hacks, IPL and firmware files, etc.
 
-Save states and game saves do not fall under this category, so please post them if they are applicable to your issue!
\ No newline at end of file
+Save states and game saves do not fall under this category, so please post them only if they are applicable to your issue!
+
+# Github Contribution Guide
+
+1. Create a frok of the repository on your own account.
```

Comment:
Should it be fork instead of frok?

Reply:
Good catch! I didn't notice. 😄 But yes it should be.